packages/compass-signtool/package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "@mongodb-js/compass-signtool",
+  "description": "Signtool to sign Compass windows artifacts",
+  "author": {
+    "name": "MongoDB Inc",
+    "email": "compass@mongodb.com"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "bugs": {
+    "url": "https://jira.mongodb.org/projects/COMPASS/issues",
+    "email": "compass@mongodb.com"
+  },
+  "homepage": "https://github.com/mongodb-js/compass",
+  "version": "0.1.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mongodb-js/compass.git"
+  },
+  "files": [
+    "dist"
+  ],
+  "license": "SSPL",
+  "main": "dist/index.js",
+  "exports": {
+    "import": "./dist/.esm-wrapper.mjs",
+    "require": "./dist/index.js",
+    "compass-signtool.exe": "./dist/compass-signtool.exe"
+  },
+  "compass:exports": {
+    ".": "./src/index.ts"
+  },
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "bootstrap": "npm run compile",
+    "prepublishOnly": "npm run compile && compass-scripts check-exports-exist",
+    "compile": "tsc -p tsconfig.json && gen-esm-wrapper . ./dist/.esm-wrapper.mjs",
+    "typecheck": "tsc -p tsconfig-lint.json --noEmit",
+    "eslint": "eslint",
+    "prettier": "prettier",
+    "lint": "npm run eslint . && npm run prettier -- --check .",
+    "depcheck": "depcheck",
+    "check": "npm run typecheck && npm run lint && npm run depcheck",
+    "check-ci": "npm run check",
+    "test": "mocha",
+    "test-cov": "nyc --compact=false --produce-source-map=false -x \"**/*.spec.*\" --reporter=lcov --reporter=text --reporter=html npm run test",
+    "test-watch": "npm run test -- --watch",
+    "test-ci": "npm run test-cov",
+    "reformat": "npm run eslint . -- --fix && npm run prettier -- --write .",
+    "build": "npm run compile && pkg package.json",
+    "postinstall": "npm run build"
+  },
+  "bin": "dist/cli.js",
+  "pkg": {
```

Comment:
are we sure we want to create an exe like this, we can assume our CI machines to have node.js installed, and we only need the signtool for wininstaller and compass, i think is 1000 times easier and safer (with less unknowns) to have this package not export any binary, and just modify the `signtool.go` that is in hadron-build, and just make it calls `npx run @mongodb-js/this-package sign`, rather than completely change the way we are doing it, and generalize it right away.